frontend/src/components/__tests__/__snapshots__/header.spec.tsx.snap

```diff
@@ -63,6 +63,13 @@ exports[`Header component matches snapshot 1`] = `
             >
               Inventory
             </a>
+            <a
+              class="makeStyles-link-16"
+              href="/feeds"
```

Comment:
Is a "feed" the same thing as a "saved search"? Or are there other types of feeds?

Reply:
Right now saved search is the only type of feed. We could consider renaming to saved searches

Reply:
Sounds good, let's keep it "feeds" for now